xpano/utils/text.h

```diff
@@ -8,6 +8,9 @@ namespace xpano::utils {
 struct Text {
   std::string name;
   std::vector<std::string> lines;
+  bool operator== (const Text& operand) {
+    return ((name == operand.name) && (lines == operand.lines));
+  }
 };
 
```

Comment:
Please add `using Texts = std::vector<Text>` and use the alias elsewhere in the code